tf2_geometry_msgs/src/tf2_geometry_msgs/__init__.py

```diff
@@ -1 +1 @@
-from tf2_geometry_msgs import *
+from .tf2_geometry_msgs import *
```

Comment:
I'm not opposed to this change, but I'm surprised it is needed.  Is it?

Reply:
there is code inside `tf2_geometry_msgs.py` that needs to run when the module is imported. I didn't find another way to do it

Reply:
All right, that's fine.  I'll approve.